tests/console/yast2_samba.pm

```diff
@@ -38,13 +38,10 @@ sub run {
         record_soft_failure 'bsc#1064405';
         send_key 'alt-c';
     }
-    send_key 'alt-f';
-
-    assert_screen([qw(yast2_ldap_configuration_general-setting_firewall yast2_still_susefirewall2)], 60);
-    if (match_has_tag 'yast2_still_susefirewall2') {
-        record_soft_failure 'bsc#1064405';
-        send_key 'alt-e';
+    if (!is_sle && !sle_version_at_least('15')) {
```

Comment:
I'm not sure if this is correct condition. sle_version_at_least always returns true if it's not sle distri (that's confusing part), so it will return false only if sle version is lower than 15, meaning, this condition will be never true. It's easier to get it if you flip it: !(is_sle || sle_version_at_least('15')). If it's not sle second condition gives true, if it's sle, then first one is already true. After negation, it's always false.

Reply:
okay, !(is_sle || sle_version_at_least('15')) is clear to me as well.